test-data/unit/semanal-types.test

```diff
@@ -180,14 +180,16 @@ x = None # type: A[B]
 y = None # type: A[Any]
 [out]
 MypyFile:1(
-  ImportFrom:1(typing, [TypeVar, Generic, Any])
+  ImportFrom:2(typing, [TypeVar, Generic, Any])
   AssignmentStmt:3(
     NameExpr(t* [__main__.t])
     TypeVarExpr:3())
   ClassDef:5(
     A
     TypeVars(
       t)
+    BaseType(
+      builtins.object)
```

Comment:
Maybe we can just update the test runner to remove the `object` base class before dumping?

Reply:
I initially didn't do this because this would skip some testing of base types, but in retrospect it now seems worth it, as we can run the same test cases with both the old and the new semantic analyzers.